src/formatDate.js

```diff
@@ -8,7 +8,59 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldDate = date.split(fromFormat[3]);
+  const newDate = [];
+
+  let year = '';
+  let month = '';
+  let day = '';
+  let oldYearFormat = '';
+  let newYearFormat = '';
+
+  for (let i = 0; i < fromFormat.length; i++) {
+    if (fromFormat[i].includes('D')) {
```

Comment:
Again, in this case, you can simply use `==DD`, because you don't have other formats with `D`

Reply:
Got it, thanks!